src/main/webapp/app/overview/visualizations/exercise-scores-chart/exercise-scores-chart.component.html

```diff
@@ -46,12 +46,12 @@ <h2 class="col-9">{{ 'artemisApp.exercise-scores-chart.title' | artemisTranslate
         >
             <ng-template #tooltipTemplate let-model="model">
                 <b>{{ model.name }}</b> <br />
-                <span>{{ model.series }}: {{ Math.max(model.value - 1, 0) }} %</span>
+                <span>{{ model.series }}: {{ Math.max(model.value, 0) }} %</span>
             </ng-template>
             <ng-template #seriesTooltipTemplate let-model="model">
                 <ng-container *ngIf="model.length">
                     <h6>{{ model[0].name }}</h6>
-                    <div *ngFor="let entry of model">{{ entry.series }}: {{ Math.max(entry.value - 1, 0) }}%</div>
+                    <div *ngFor="let entry of model; let i = index">{{ entry.series }}: {{ Math.max(entry.value, 0) }}%</div>
```

Comment:
Where do you use the index?

Reply:
Good catch - I don't. Removed it in the latest commit.